noir/noir-repo/compiler/noirc_evaluator/src/ssa/opt/inlining.rs

```diff
@@ -951,7 +1076,7 @@ mod test {
         builder.switch_to_block(join_block);
         builder.terminate_with_return(vec![join_param]);
 
-        let ssa = builder.finish().inline_functions();
+        let ssa = builder.finish().inline_functions(f64::INFINITY);
```

Comment:
Would be nice to have some units tests showing an aggressiveness of `0` and `NEG_INFINITY`

Reply:
Added two tests